flink-python/src/main/java/org/apache/flink/client/python/PythonShellParser.java

```diff
@@ -192,6 +192,17 @@ private static void printYarnHelp() {
         formatter.printHelp(" ", YARN_OPTIONS);
     }
 
+    private static void printYarnWarnMessage() {
```

Comment:
I do not think we only need to print the warning message for python parser.

IIUC, when the `FlinkYarnSessionCli` is actived(session mode, perjob), we need to print the warning messages and suggest the users to use "-t/--target" instead.